moped-database/migrations/1663082078687_adds_pd_subphases_and_updates_phase_description/down.sql

```diff
@@ -0,0 +1,11 @@
+-- reverts add subphase "Preliminary Engineering Report" to the "Preliminary Engineering" phase
```

Comment:
Thanks for deleting these by column values and not by the ID they get in a given environment - great pattern! 👍